service-loadbalancer/service_loadbalancer.go

```diff
@@ -418,6 +421,13 @@ func healthzServer() {
 			}
 		}
 	})
+
+	// handler for not matched traffic
```

Comment:
it's ok to put all handlers in one function, but please rename to something like registerHandlers (currently called healthzServer)